tmva/sofie/src/RModel.cxx

```diff
@@ -339,6 +339,10 @@ namespace SOFIE{
             fGC += "std::vector<int64_t> fTensor_" + i.first  + " = std::vector<int64_t>(" + std::to_string(length) + ");\n";
             fGC += "int64_t * tensor_" + i.first + " = fTensor_" + i.first  + ".data();\n";
          }
+         if (i.second.type == ETensorType::BOOL){
+            fGC += "std::vector<bool> fTensor_" + i.first  + " = std::vector<bool>(" + std::to_string(length) + ");\n";
+            fGC += "bool * tensor_" + i.first + " = fTensor_" + i.first  + ".data();\n";
+         }
```

Comment:
Initialize a Boolean array instead, as the `.data` method does not exist in boolean vectors.